white-list

```diff
@@ -30,3 +30,4 @@ ui.argocd.yourorganization.net:443
 your-kubernetes-cluster-addr
 yourorganization.oktapreview.com
 yourorganization.oktapreview.com:443
+example-OIDC-provider.com
```

Comment:
I noticed that this URL was failing when running `make lint-docs` so I've added it to the allowlist